packages/create-sitecore-jss/src/common/args/base.ts

```diff
@@ -29,6 +29,10 @@ export type BaseArgs = {
    * Default values will be used
    */
   yes?: boolean;
+  /**
+   * Pre-push hook for linting check
+   */
+  prePush?: boolean;
```

Comment:
Do you think "prePush" is too generic? I kinda liked your original name with "Hook" at the end. 
```suggestion
  prePushHook?: boolean;
```

Reply:
Honestly, I was a bit confused about the naming. I thought `prePushHook` would be long and hard to remember but i guess it makes sense to add hook in the end for more clarity.